spec/lib/submission_builder/ty2024/states/nc/documents/d400_spec.rb

```diff
@@ -33,6 +37,10 @@
         expect(xml.document.at('FAGIPlusAdditions').text).to eq "10000"
         expect(xml.document.at('NCStandardDeduction').text).to eq standard_deduction.to_s
         expect(xml.document.at('NCAGIAddition').text).to eq nc_agi_addition.to_s
+        expect(xml.document.at('NCAGISubtraction').text).to eq nc_agi_subtraction.to_s
+        expect(xml.document.at('NCTaxableInc').text).to eq nc_agi_subtraction.to_s # TODO: validate assumption that we only have full year residents at this stage
```

Comment:
is this TODO for now or later?

Reply:
oops it's for now

Reply:
ok i checked and it is true. comment removed!